src/api/public/apidocs-new/paths/request_id.yaml

```diff
@@ -73,6 +73,9 @@ post:
           after the last review.
 
         - `cancelapproval`: Reset the approval of a request
+
+        - `superseded_by`: Id of the request superseding this one. Required for
```

Comment:
`superseded_by` is not a command for this endpoint. This option should be documented under the `changestate` command.